webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+  module: {
```

Comment:
- [ ] Your webpack configuration is missing the `webpack dev-server`. Kindly ensure to fix this in your webpack configuration to enable the application to be run on a server

Reply:
This issue is resolved well! Excellent work! :champagne: